config/locales/email.yml

```diff
@@ -0,0 +1,34 @@
+en:
+  nfg_ui:
+    email:
+      company:
+        name: "Network for Good"
+        address: "Network for Good | 1140 Connecticut Avenue NW, Suite 700, Washington, District of Columbia 20036"
+      images:
+        logo: "https://identity.networkforgood.org/images/nfg-logo.png"
+        alt:
+          caret: ">"
+          logo: "Network for Good"
+          social:
+            facebook: "Network for Good on Facebook"
+            twitter: "Network for Good on Twitter"
+            linkedin: "Network for Good on LinkedIn"
+            youtube: "Network for Good on YouTube"
+            google_plus: "Network for Good on Google+"
```

Comment:
I think we can remove google plus since we just removed it on evo campaign pages.